### PR TITLE
supervisor-edit-subject

### PR DESCRIPTION
**Pull request by hienbx94**
I have created pull request. Please check it

### CODE REVIEW

**Comment by trantube94** [comment]
OK

**Comment by ttuan** [comment]
OK

File: app/controllers/admin/subjects_controller.rb [hunk 1]
```diff
@@ -14,6 +14,20 @@ def create
...
+      flash.now[:success] = t "subjects.subject_updated"
+      redirect_to admin_subjects_path 
+    else
+      render "edit"
+    end
+  end
+
   private
   def subject_params
     params.require(:subject).permit :name, :description, :day_work,
```
Comment (ThuBM) [inline app/controllers/admin/subjects_controller.rb]: params must permit :id

**Comment by ThuBM** [comment]
OK

**Pull request merged**